### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.maturityStable_vmFG.maturityBadge_GUVd)

### ISSUE THREAD

**Issue by WilliamBerryiii** [microsoft/hve-core#1917]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[HVE Core](https://microsoft.github.io/hve-core/)
#### Element path
.collectionCard_PZk5:nth-child(10) > .collectionHeader_L9pK > .maturityStable_vmFG.maturityBadge_GUVd
#### Snippet
`<span class="maturityBadge_GUVd maturityStable_vmFG">Stable</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.56 (foreground color: #009400, background color: #e6f6e6, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 149.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
